From 40dd3ded350214fb64ff53faa5d2fc210d20afde Mon Sep 17 00:00:00 2001
From: Michal Simek <monstr@monstr.eu>
Date: Sun, 2 Oct 2011 15:22:21 +0200
Subject: [PATCH] axienet: Do not setup app2 in start_xmit

app2 is clear in bd_init function and start_xmit_done
for reused BDs.

Signed-off-by: Michal Simek <monstr@monstr.eu>
---
 drivers/net/ethernet/xilinx/xilinx_axienet_main.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/net/ethernet/xilinx/xilinx_axienet_main.c b/drivers/net/ethernet/xilinx/xilinx_axienet_main.c
index b329349..9c966cb 100755
--- a/drivers/net/ethernet/xilinx/xilinx_axienet_main.c
+++ b/drivers/net/ethernet/xilinx/xilinx_axienet_main.c
@@ -780,7 +780,6 @@ static int axienet_start_xmit(struct sk_buff *skb, struct net_device *ndev)
 			/* Tx Partial Checksum Offload Enabled */
 			cur_p->app0 |= 1;
 			cur_p->app1 = (csum_start_off << 16) | csum_index_off;
-			cur_p->app2 = 0;  /* initial checksum seed */
 		}
 	} else if (skb->ip_summed == CHECKSUM_UNNECESSARY) {
 		cur_p->app0 |= 2; /* Tx Full Checksum Offload Enabled */
-- 
1.7.0.4

